packages/cli/src/commands/ui/create/atomic.ts

```diff
@@ -0,0 +1,74 @@
+import {Command} from '@oclif/command';
+import {
+  buildAnalyticsFailureHook,
+  buildAnalyticsSuccessHook,
+} from '../../../hooks/analytics/analytics';
+import {
+  Preconditions,
+  IsAuthenticated,
+  HasNecessaryCoveoPrivileges,
+} from '../../../lib/decorators/preconditions/';
+import {
+  createApiKeyPrivilege,
+  impersonatePrivilege,
+} from '../../../lib/decorators/preconditions/platformPrivilege';
+import {appendCmdIfWindows} from '../../../lib/utils/os';
+import {spawnProcess} from '../../../lib/utils/process';
+
+interface AtomicArguments {
+  name: string;
+}
+
+export default class Atomic extends Command {
+  public static description =
+    "Create a Coveo Headless-powered search page with Coveo's own Atomic framework. See <https://docs.coveo.com/atomic> and <https://docs.coveo.com/headless>.";
+
+  public static args = [
+    {
+      name: 'name',
+      description: 'The name of the application to create.',
+      required: true,
+    },
+  ];
+
+  @Preconditions(
```

Comment:
// TODO: Add the `@Trackable()` decorator from this pr: https://github.com/coveo/cli/pull/523